src/fty_alert_actions.cc

```diff
@@ -845,6 +852,30 @@ s_handle_stream_deliver_asset(fty_alert_actions_t *self, fty_proto_t **asset_p,
     }
 }
 
+//  --------------------------------------------------------------------------
+//  Handle incoming metric through stream
+static void
+s_handle_stream_deliver_metric(fty_alert_actions_t *self, fty_proto_t *message)
+{
+    assert (self);
+    assert (message);
+    if (streq (fty_proto_name(message), "rackcontroller-0") && streq (fty_proto_type(message), "notification_and_action.global"))
+    {
+        int allowNotificationAndAction = std::stoi(fty_proto_value(message));
```

Comment:
you should put this into a try/catch block, since it may throw an exception (either invalid or out of range)